examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/ui/screens/paywallfooter/PaywallFooterScreen.kt

```diff
@@ -0,0 +1,96 @@
+package com.revenuecat.paywallstester.ui.screens.paywallfooter
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.AlertDialog
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.revenuecat.paywallstester.ui.screens.paywall.PaywallScreenState
+import com.revenuecat.paywallstester.ui.screens.paywall.PaywallScreenViewModel
+import com.revenuecat.paywallstester.ui.screens.paywall.PaywallScreenViewModelImpl
+import com.revenuecat.purchases.ui.revenuecatui.PaywallFooter
+import com.revenuecat.purchases.ui.revenuecatui.PaywallViewOptions
+
+@Composable
+fun PaywallFooterScreen(
+    viewModel: PaywallScreenViewModel = viewModel<PaywallScreenViewModelImpl>(),
+) {
+    Column(
+        modifier = Modifier.fillMaxSize(),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Center,
+    ) {
+        when (val state = viewModel.state.collectAsState().value) {
+            is PaywallScreenState.Loading -> {
+                Text(text = "Loading...")
+            }
+            is PaywallScreenState.Error -> {
+                Text(text = "Error: ${state.errorMessage}")
+            }
+            is PaywallScreenState.Loaded -> {
+                PaywallFooter(
+                    options = PaywallViewOptions.Builder()
+                        .setOffering(state.offering)
```

Comment:
Wait hmmm does this mean that in order to use `PaywallFooter`, users need to load the offering themselves?

Reply:
Nope, this setter is optional. If you don't set it, we will load the `current` offering by default